### ISSUE TITLE
Generically disable local IP addresses in harvester

### ISSUE THREAD

**Issue by oliverhausler** [jitsi/jitsi-videobridge#677]
The following configuration options would make sense in `sip-communicator.properties`:

1. Disallow local IPs generically, similar to `org.ice4j.ipv6.DISABLED`, e.g. `org.ice4j.ipLocal.DISABLED`
2. Allow wildcards in `org.ice4j.ice.harvest.BLOCKED_ADDRESSES` to block an address range, e.g. `10.*.*.*` for blocking all local IPs beginning with `10.`.

Reason:

When Jitsi is deployed using k8s or similar, we may want to exclude local IPs from being harvested, but the IP the pod receives isn't known at the time of configuration. Excluding an address range would solve the problem of not having to dynamically patch the file during startup.

**Comment by bgrozev** [comment]
I agree that these make sense. I think they would be a welcome contribution to ice4j. The first one should be trivial to do (by mirroring the logic for `org.ice4j.ice.harvest.DISABLE_LINK_LOCAL_ADDRESSES` for SITE_LOCAL). The second one will be a little trickier because we store a list of `InetAddress`. I would suggest to use CIDR notation instead of wildcards (e.g. `10.0.0.0/8` for your example).

**Comment by ludovicm67** [comment]
Any update on this? I have some issues and need to be able to disable local IP addresses (particularly the `10.0.0.0/8` range).

**Comment by bgrozev** [comment]
@ludovicm67 No progress on our end, this is not on our roadmap (but contributions are welcome).
